docs/react/README.md

```diff
@@ -90,9 +204,39 @@ Included components:
 
 Learn more about the available components [here](./components.md).
 
```

Comment:
we should define the `DataGrid`, `Column`, and `Empty` components and whatever props they take, or if these are defined elsewhere already, then we should link to them directly